x-pack/plugins/security_solution/server/lib/risk_engine/risk_engine_data_client.ts

```diff
@@ -270,7 +267,29 @@ export class RiskEngineDataClient {
         indexPatterns,
       });
 
-      await this.initializeWriter(namespace, indexPatterns.alias);
+      await createIndex({
+        esClient,
+        logger: this.options.logger,
+        options: {
+          index: getRiskScoreLatestIndex(namespace),
+          mappings: mappingFromFieldMap(riskScoreFieldMap, 'strict'),
```

Comment:
> In the case of a latest transform, the mappings are never deduced. If dynamic mappings for the destination index are undesirable, use the [Create index API](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-create-index.html) prior to starting the transform.

Reply:
The problem with reusing index templates(which will be great). Our index template for DS contain ILM policy, but
> [ILM as destination index may cause duplicated documents](https://www.elastic.co/guide/en/elasticsearch/reference/current/transform-limitations.html#transform-ilm-destination)

>[ILM](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-lifecycle-management.html) is not recommended to use as a transform destination index. Transforms update documents in the current destination, and cannot delete documents in the indices previously used by ILM. This may lead to duplicated documents when you use transforms combined with ILM in case of a rollover.

Reply:
Understood, thanks for looking into this! I appreciate it.